uiowa/uiowa
# [ITS] Alert updates

Resolves https://github.com/uiowa/uiowa/issues/7360. 

# How to test

```
ddev blt frontend && ddev blt ds --site=its.uiowa.edu && ddev drush @its.local uli /testing-new-alert-thing
```
- Confirm sidebar filters are working
- Confirm summary is hidden on https://its.uiowa.ddev.site/node/add/alert
- Add some updates to an alert
  - Confirm that you are unable to drag to reorder the alerts
  - Confirm that they are sorted chronologically upon save, with the most recent on top on page display
  - Confirm that when re-editing the alert, the updates appear ordered
